src/main/java/com/github/netelli/ProductsDAO.java

```diff
@@ -90,7 +94,7 @@ public void close() throws Exception {
                 connection.close();
             }
         } catch (Exception e) {
-            System.out.println("Error while closing connection. " + e.getMessage());
+            logger.error("Error while closing connection. " + e.getMessage());
```

Comment:
tip: when logging exception use pattern logger.error(msg, exceptionObject)

so in this case it will look like:
logger.error("Error while closing connection. " + e.getMessage(), e);